molecule/packages-dynamic/group_vars/all

```diff
@@ -8,4 +8,4 @@ pulp_install_plugins:
 pulp_settings:
   secret_key: secret
   content_origin: "https://{{ ansible_fqdn }}"
-pulp_pkg_repo: "https://yum.theforeman.org/pulpcore/3.6/el{{ ansible_distribution_major_version }}/$basearch/"
+pulp_pkg_repo: "https://yum.theforeman.org/pulpcore/3.7/el{{ ansible_distribution_major_version }}/$basearch/"
```

Comment:
maybe we need a variable for it

Reply:
> the CI is failing due to dynaconf (I think we need a package with the new version)

Filed (https://projects.theforeman.org/issues/30963) and made a separate commit for robustness.